src/auth/digest/eDirectory/ldap_backend.cc

```diff
@@ -396,7 +409,7 @@ ldapconnect(void)
         }
         if (use_tls) {
 #ifdef LDAP_OPT_X_TLS
-            if ((version == LDAP_VERSION3) && (ldap_start_tls_s(ld, nullptr, nullptr) == LDAP_SUCCESS)) {
+            if ((version == LDAP_VERSION3) && (ldap_start_tls_s(ld, nullptr, nullptr) != LDAP_SUCCESS)) {
```

Comment:
> PR description: An incorrect comparison made LDAP-over-SSL (LDAPS) always fail

Ouch. This probably warrants a release notes mention OR, combined with other bugs fixed for this helper in this and other PR, helper removal from master/v8.